pages/faq.tsx

```diff
@@ -36,7 +36,7 @@ export default function Faqs(props: FaqsProps) {
   const [filteredQuestions, handleSubmitKeywords, urlParams, filterItems] =
     useSearch(
       faq,
-      ["pertanyaan", "jawaban"],
+      ["kategori_pertanyaan", "pertanyaan", "jawaban"],
```

Comment:
My personal take, if you have arguments more than 3, wrap it into an object.
It will be easier for your client to know what is the argument for.